e2e/fixtures/setup.ts

```diff
@@ -77,6 +77,17 @@ export const test = base.extend<Fixtures>({
       ),
     );
   },
+  page: async ({ page }, use) => {
+    await page.route("**/*", async (route, request) => {
+      // Speed up the tests by blocking requests such as the fonts on assets.ubuntu.com and icons from charmhub.io.
+      if (["media", "font", "image"].includes(request.resourceType())) {
+        await route.abort();
```

Comment:
I wonder if we should display a message when we block a request (like I did here: https://github.com/canonical/juju-dashboard/actions/runs/16979288192/job/48135628223?pr=2046#step:8:414). It'd be more noisy, but it might help us in the future if we were expecting an asset to be loaded.

Reply:
Done.